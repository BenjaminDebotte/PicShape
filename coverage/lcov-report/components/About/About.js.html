<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components/About/About.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\index.html">All files</a> / <a href="index.html">components/About</a> About.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">37.5% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/8</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">66.67% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>2/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">37.5% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/8</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49
50
51
52
53
54
55
56
57
58
59
60
61
62
63
64
65
66
67
68
69
70
71
72
73
74
75
76
77
78
79
80
81
82
83
84
85
86
87
88
89
90
91
92
93
94
95
96
97
98
99
100
101
102
103
104
105
106
107
108
109
110
111
112
113
114
115
116
117</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React, { Component } from 'react';
import './About.css';
&nbsp;
import baseImg from './548x430.jpg';
import convertedImg from './548x430Shaped.jpg';
&nbsp;
export class About extends Component {
&nbsp;
  constructor(props) {
      super(props);
      this.state = {
        changeImg: true
    }
  }
&nbsp;
<span class="fstat-no" title="function not covered" >  _c</span>hangeImage(input) {
<span class="cstat-no" title="statement not covered" >      if (this.state.changeImg)</span>
      {
<span class="cstat-no" title="statement not covered" >          document.getElementById("convertedImg").src = convertedImg;</span>
<span class="cstat-no" title="statement not covered" >          this.setState({changeImg: false});</span>
      }
      else
      {
<span class="cstat-no" title="statement not covered" >          document.getElementById("convertedImg").src = baseImg;</span>
<span class="cstat-no" title="statement not covered" >          this.setState({changeImg: true});</span>
      }
  }
&nbsp;
render() {
    return (
&nbsp;
&lt;div className="ui raised segment"&gt;
  &lt;div className="ui equal width grid "&gt;
    &lt;div className="column"&gt;
      &lt;div className="ui"&gt;
      &lt;img className="ui fluid rounded image myImg" onClick={this._changeImage.bind(this)} id="convertedImg" src={baseImg} alt="" onMouseOver=""/&gt;
      &lt;/div&gt;
    &lt;/div&gt;
    &lt;div className="column"&gt;
      &lt;div className="ui segment"&gt;
        &lt;h2&gt;ABOUT US&lt;/h2&gt;
&nbsp;
      &lt;p&gt;The Picshape project is composed of a front-end, back-end and an Android application.&lt;/p&gt;
      &lt;p&gt;Picshape use a Node.js back-end hosting the API for image transformation and a front-end using React and Redux.
      It is a 3rd year project part of 'PicShape',
      a cloud image converter using fogleman primitive project.&lt;/p&gt;
&nbsp;
      &lt;h2&gt;Android Application&lt;/h2&gt;
      &lt;ul&gt;
        &lt;li&gt;Aliquam venenatis leo et orci.&lt;/li&gt;
        &lt;li&gt;Pellentesque eleifend vulputate massa.&lt;/li&gt;
        &lt;li&gt;Vivamus eleifend sollicitudin eros.&lt;/li&gt;
        &lt;li&gt;Maecenas vitae nunc.&lt;/li&gt;
        &lt;/ul&gt;
        &lt;/div&gt;
    &lt;/div&gt;
&nbsp;
&nbsp;
    &lt;div className="equal width row"&gt;
      &lt;div className="column"&gt;
          &lt;div className="ui "&gt;
            &lt;h4 className="ui center aligned icon header"&gt;
              &lt;i className="circular users icon"&gt;&lt;/i&gt;
                  &lt;/h4&gt;
            &lt;/div&gt;
      &lt;/div&gt;
    &lt;/div&gt;
&nbsp;
    &lt;div className="equal width row"&gt;
        &lt;div className="column"&gt;
          &lt;div className="ui raised segment"&gt;
          &lt;img className="ui medium circular rounded image" src={require('./team-member.gif')} alt="" /&gt;
                &lt;p className="item"&gt;BenJ&lt;/p&gt;
                &lt;p className="item"&gt;Job Title Here&lt;/p&gt;
          &lt;/div&gt;
          &lt;/div&gt;
        &lt;div className="column"&gt;
        &lt;div className="ui raised segment"&gt;
        &lt;img className="ui medium circular rounded image" src={require('./team-member.gif')} alt="" /&gt;
              &lt;p className="item"&gt;Guillaume&lt;/p&gt;
              &lt;p className="item"&gt;Job Title Here&lt;/p&gt;
        &lt;/div&gt;
        &lt;/div&gt;
        &lt;div className="column"&gt;
        &lt;div className="ui raised segment"&gt;
        &lt;img className="ui medium circular rounded image" src={require('./team-member.gif')} alt="" /&gt;
              &lt;p className="item"&gt;ShowTime&lt;/p&gt;
              &lt;p className="item"&gt;Job Title Here&lt;/p&gt;
        &lt;/div&gt;
        &lt;/div&gt;
        &lt;div className="column"&gt;
        &lt;div className="ui raised segment"&gt;
        &lt;img className="ui medium circular rounded image" src={require('./team-member.gif')} alt="" /&gt;
              &lt;p className="item"&gt;Philibert&lt;/p&gt;
              &lt;p className="item"&gt;Job Title Here&lt;/p&gt;
        &lt;/div&gt;
        &lt;/div&gt;
        &lt;div className="column"&gt;
        &lt;div className="ui raised segment"&gt;
        &lt;img className="ui medium circular rounded image" src={require('./team-member.gif')} alt="" /&gt;
              &lt;p className="item"&gt;Antho&lt;/p&gt;
              &lt;p className="item"&gt;Job Title Here&lt;/p&gt;
        &lt;/div&gt;
        &lt;/div&gt;
      &lt;/div&gt;
  &lt;/div&gt;
&nbsp;
&lt;/div&gt;
&nbsp;
&nbsp;
&nbsp;
    );
  }
}
&nbsp;
export default About;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Thu Mar 09 2017 10:50:14 GMT+0100 (Romance Standard Time)
</div>
</div>
<script src="..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\sorter.js"></script>
</body>
</html>
