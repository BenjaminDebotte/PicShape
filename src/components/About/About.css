.about {
  text-align: center;
}

.myImg {
    border-radius: 5px;
    cursor: pointer;
    transition: 0.3s;
}

.myImg:hover {opacity: 0.7;}

#about-us{display:block; width:100%; line-height:1.6em;}
#about-us h2{margin-bottom:10px; padding:0;} /* DEMO USE ONLY - CAN BE DELETED */

#about-us #about-intro{margin-bottom:30px;}
#about-us #about-intro .imgholder{padding:9px;}
#about-us #about-intro h2{margin-bottom:auto;}
#about-us #about-intro p{}
#about-us #about-intro p:first-child{}
#about-us #about-intro ul{}
#about-us #about-intro ul li{margin-top:5px;}
#about-us #about-intro ul li:first-child{margin-top:0;}

#about-us #client_logos{margin-bottom:50px; padding:40px 0; border:solid #CCCCCC; border-width:1px 0;}
#about-us #client_logos ul{margin:0; padding:0; list-style:none;}
#about-us #client_logos ul li{}
#about-us #client_logos ul li:first-child{margin-left:0;}
#about-us #client_logos ul li img{}

#about-us #team{}
#about-us #team ul{margin:0; padding:0; list-style:none; text-align:center;}
#about-us #team ul li{margin-bottom:10px; padding-bottom:0px; border-bottom:1px solid #CCCCCC;}
#about-us #team ul li .figure{}
#about-us #team ul li .figure img{padding:9px; border:1px solid #CCCCCC;}
#about-us #team ul li .figure .figcaption{}
#about-us #team ul li p{margin:15px 0 0 0; padding:0;}
#about-us #team ul li p:first-child{}
#about-us #team ul li .figure .figcaption .team_name{font-size:18px;}
#about-us #team ul li .figure .figcaption .team_title{margin:0; font-size:11px; line-height:normal;}

.one_half,
.one_third, .two_third,
.one_quarter, .two_quarter, .three_quarter,
.one_fifth, .two_fifth, .three_fifth, .four_fifth{display:inline-block; float:left; margin:0 0 0 20px; list-style:none;}

.one_third, .two_third{margin:0 0 0 30px;}

.first,
.one_half:first-child,
.one_third:first-child, .two_third:first-child,
.one_quarter:first-child, .two_quarter:first-child, .three_quarter:first-child,
.one_fifth:first-child, .two_fifth:first-child, .three_fifth:first-child, .four_fifth:first-child{margin-left:0;}

.two_half, .three_third, .four_quarter, .five_fifth{display:block; width:100%; clear:both;}

/*----------------------------------------------Half Grid-------------------------------------*/

.one_half{width:470px;}

/*----------------------------------------------Third Grid-------------------------------------*/

.one_third{width:300px;}
.two_third{width:630px;}

/*----------------------------------------------Quarter Grid-------------------------------------*/

.one_quarter{width:225px;}
.two_quarter{width:470px;}
.three_quarter{width:715px;}

/*----------------------------------------------Fifth Grid-------------------------------------*/

.one_fifth{width:176px;}
.two_fifth{width:372px;}
.three_fifth{width:510px;}
.four_fifth{width:764px;}

html{overflow-y:scroll;}
body{margin:0;padding:0;font-size:13px;font-family:Georgia,"Times New Roman",Times,serif;color:#919191;background-color:#232323;}
.justify{text-align:justify;}
.bold{font-weight:bold;}
.center{text-align:center;}
.right{text-align:right;}
.nostart{margin:0;padding:0;list-style:none;}
.hidden{display:none;}
.clear:after{content:".";display:block;height:0;clear:both;visibility:hidden;line-height:0;}
.clear{display:inline-block;clear:both;}html[xmlns]
.clear{display:block;}* html
.clear{height:1%;}
a{outline:none;text-decoration:none;}
.fl_left{float:left;}
.fl_right{float:right;}
img{margin:0;padding:0;border:none;line-height:normal;vertical-align:middle;}
.imgholder,
.imgl,
.imgr{padding:4px;border:1px solid #D6D6D6;text-align:center;}
.imgl{float:left;margin:0 15px 15px 0;clear:left;}
.imgr{float:right;margin:0 0 15px 15px;clear:right;}
address,article,aside,figcaption,figure,footer,header,hgroup,nav,section{display:block;margin:0;padding:0;}
div.wrapper{display:block;width:100%;margin:0;padding:0;text-align:left;}
.row1,.row1 a{color:#C0BAB6;background-color:#333333;}
.row2{color:#979797;background-color:rgba(255, 255, 255, 0);}.row2 a{color:#FF9900;background-color:#FFFFFF;}
.row3,.row3 a{color:#919191;background-color:#232323;}
div.wrapper h1,div.wrapper h2,div.wrapper h3,div.wrapper h4,div.wrapper h5,div.wrapper h6{margin:0 0 15px 0;padding:0;font-size:20px;font-weight:normal;line-height:normal;}
form,fieldset,legend{margin:0;padding:0;border:none;}
legend{display:none;}
input,textarea,select{font-size:12px;font-family:Georgia,"Times New Roman",Times,serif;}
#header,#container,#footer{display:block;width: 983px;margin:0 auto;}
#header{padding:20px 0;}
#header .fl_left h1,
#header .fl_left h2{margin:0;padding:0;font-weight:normal;text-transform:none;}
#header .fl_left h1{font-size:36px;}
#header .fl_left h2{font-size:13px;}
#header #topnav{display:block;float:right;margin:0;padding:20px;list-style:none;color:#C0BAB6;background-color:#232323;-webkit-border-radius:10px;-moz-border-radius:10px;border-radius:10px;}
#header #topnav li{display:inline;margin-right:25px;text-transform:uppercase;}
#header #topnav li.last{margin-right:0;}
#header #topnav li a{color:#C0BAB6;background-color:#232323;}
#header #topnav li a:hover,#header #topnav li.active a{color:#FF9900;background-color:#232323;}
#container{padding: 0px 0;}
#container a{background-color:transparent;}
#container #portfolio,#container #gallery{margin-bottom:30px;}
#container .pagination{display:block;width:100%;text-align:center;clear:both;font-family:Verdana,Geneva,sans-serif;}
#container .pagination ul{margin:0;padding:0;list-style:none;}
#container .pagination li{display:inline;}
#container .pagination strong{font-weight:normal;}
#container .pagination .next{margin:0;}
#container .pagination a,#container .pagination .current strong,#container .pagination .splitter strong{display:inline-block;padding:2px 6px;margin:0 2px 0 0;}
#container .pagination a{border:1px solid #DFDFDF;}#footer{padding:20px 0;}#footer p{margin:0;padding:0;}
