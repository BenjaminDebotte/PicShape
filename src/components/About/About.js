import React, { Component } from 'react';
import './About.css';

import baseImg from './548x430.jpg';
import convertedImg from './548x430Shaped.jpg';

class About extends Component {

  constructor(props) {
      super(props);
      this.state = {
        changeImg: true
    }
  }

  _changeImage(input) {
      if (this.state.changeImg)
      {
          document.getElementById("convertedImg").src = convertedImg;
          this.setState({changeImg: false});
      }
      else
      {
          document.getElementById("convertedImg").src = baseImg;
          this.setState({changeImg: true});
      }
  }

render() {
    return (
      <div className="wrapper row2">
      <div id="container" className="clear">

    <div id="about-us" className="clear">

      <div id="about-intro" className="clear">
        <div className="three_fifth first"><img className="imgholder myImg" onClick={this._changeImage.bind(this)} id="convertedImg" src={baseImg} alt="" onMouseOver=""/>
       </div>
        <div className="two_fifth">
          <h2>ABOUT US</h2>

          <p>The Picshape project is composed of a front-end, back-end and an Android application.</p>
          <p>Picshape use a Node.js back-end hosting the API for image transformation and a front-end using React and Redux.
          It is a 3rd year project part of 'PicShape',
          a cloud image converter using fogleman primitive project.</p>


          <h2>Android Application</h2>
          <ul>
            <li>Aliquam venenatis leo et orci.</li>
            <li>Pellentesque eleifend vulputate massa.</li>
            <li>Vivamus eleifend sollicitudin eros.</li>
            <li>Maecenas vitae nunc.</li>
            <li>Ut pretium odio eu nisi.</li>
            <li>Nam condimentum mi id magna.</li>
            <li>Pellentesque consectetuer, felis vel rhoncus.</li>
          </ul>
        </div>
      </div>

        <div id="team">
        <h2>The Team</h2>
        <ul className="clear">
          <li className="one_fifth first">
            <div className="figure"><img src={require('./team-member.gif')} alt="" />
              <div className="figcaption">
                <p className="team_name">BenJ</p>
                <p className="team_title">Job Title Here</p>
              </div>
            </div>
          </li>
          <li className="one_fifth">
            <div className="figure"><img src={require('./team-member.gif')} alt="" />
              <div className="figcaption">
                <p className="team_name">Chautime</p>
                <p className="team_title">Job Title Here</p>
              </div>
            </div>
          </li>
          <li className="one_fifth">
            <div className="figure"><img src={require('./team-member.gif')} alt="" />
              <div className="figcaption">
                <p className="team_name">Philibert</p>
                <p className="team_title">Job Title Here</p>
              </div>
            </div>
          </li>
          <li className="one_fifth">
            <div className="figure"><img src={require('./team-member.gif')} alt="" />
              <div className="figcaption">
                <p className="team_name">Gaillaume</p>
                <p className="team_title">Job Title Here</p>
              </div>
            </div>
          </li>
          <li className="one_fifth">
            <div className="figure"><img src={require('./team-member.gif')} alt="" />
              <div className="figcaption">
                <p className="team_name">Antho</p>
                <p className="team_title">Job Title Here</p>
              </div>
            </div>
          </li>
        </ul>
      </div>
      </div>
  </div>
</div>
    );
  }
}

export default About;
