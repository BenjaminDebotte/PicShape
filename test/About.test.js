import React from 'react';
import  {About} from '../src/components/About/About';
import renderer from 'react-test-renderer';

describe('components', function() {
  describe('<About />', function() {
    it('renders correctly', function() {
      var tree = renderer.create(<About />).toJSON();
      expect(tree).toMatchSnapshot();
        });
  });
});
