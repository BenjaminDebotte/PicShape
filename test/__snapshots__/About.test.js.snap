exports[`components <About /> renders correctly 1`] = `
<div
  className="ui raised segment">
  <div
    className="ui equal width grid ">
    <div
      className="column">
      <div
        className="ui">
        <img
          alt=""
          className="ui fluid rounded image myImg"
          id="convertedImg"
          onClick={[Function bound _changeImage]}
          onMouseOver=""
          src="test-file-stub" />
      </div>
    </div>
    <div
      className="column">
      <div
        className="ui segment">
        <h2>
          ABOUT US
        </h2>
        <p>
          The Picshape project is composed of a front-end, back-end and an Android application.
        </p>
        <p>
          Picshape use a Node.js back-end hosting the API for image transformation and a front-end using React and Redux. It is a 3rd year project part of \'PicShape\', a cloud image converter using fogleman primitive project.
        </p>
        <h2>
          Android Application
        </h2>
        <ul>
          <li>
            Aliquam venenatis leo et orci.
          </li>
          <li>
            Pellentesque eleifend vulputate massa.
          </li>
          <li>
            Vivamus eleifend sollicitudin eros.
          </li>
          <li>
            Maecenas vitae nunc.
          </li>
        </ul>
      </div>
    </div>
    <div
      className="equal width row">
      <div
        className="column">
        <div
          className="ui ">
          <h4
            className="ui center aligned icon header">
            <i
              className="circular users icon" />
          </h4>
        </div>
      </div>
    </div>
    <div
      className="equal width row">
      <div
        className="column">
        <div
          className="ui raised segment">
          <img
            alt=""
            className="ui medium circular rounded image"
            src="test-file-stub" />
          <p
            className="item">
            BenJ
          </p>
          <p
            className="item">
            Job Title Here
          </p>
        </div>
      </div>
      <div
        className="column">
        <div
          className="ui raised segment">
          <img
            alt=""
            className="ui medium circular rounded image"
            src="test-file-stub" />
          <p
            className="item">
            Guillaume
          </p>
          <p
            className="item">
            Job Title Here
          </p>
        </div>
      </div>
      <div
        className="column">
        <div
          className="ui raised segment">
          <img
            alt=""
            className="ui medium circular rounded image"
            src="test-file-stub" />
          <p
            className="item">
            ShowTime
          </p>
          <p
            className="item">
            Job Title Here
          </p>
        </div>
      </div>
      <div
        className="column">
        <div
          className="ui raised segment">
          <img
            alt=""
            className="ui medium circular rounded image"
            src="test-file-stub" />
          <p
            className="item">
            Philibert
          </p>
          <p
            className="item">
            Job Title Here
          </p>
        </div>
      </div>
      <div
        className="column">
        <div
          className="ui raised segment">
          <img
            alt=""
            className="ui medium circular rounded image"
            src="test-file-stub" />
          <p
            className="item">
            Antho
          </p>
          <p
            className="item">
            Job Title Here
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
